### PR TITLE
Setup for subjects mono icons

### PR DESCRIPTION
**Pull request by burakukula**
screen shot 2017-07-21 at 16 11 56
screen shot 2017-07-21 at 16 12 09

### CODE REVIEW

File: scripts/tasks/svgs-generate.js [hunk 1]
```diff
@@ -46,6 +47,25 @@ module.exports = function(gulp, plugins, consts) {
...
+        symbol: {
+          sprite: '../subjects-mono-icons.js'
+        }
+      },
+      shape: {
+        id: {
+          generator: 'icon-subject-mono-%s'
+        },
+        transform: ['svgo', {
+          custom: svgSymbolCleanUp.bind(null, {removeClass: false})
```
Comment (kdzwinel) [inline scripts/tasks/svgs-generate.js]: removeClass: true ?

**Comment by kdzwinel** [comment]
Please update the main docs page - https://github.com/brainly/style-guide/blob/master/src/docs/pages/index.jsx

File: scripts/tasks/svgs-generate.js (hunk 1 above)
Comment (burakukula) [inline scripts/tasks/svgs-generate.js]: true.thx

**Comment by burakukula** [comment]
@kdzwinel please have a look, I updated stuff.

File: src/components/subject-mono-icons/pages/subject-mono-icons.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import DocsBlock from 'DocsBlock';
+import ContrastBox from 'ContrastBox';
```
Comment (kdzwinel) [inline src/components/subject-mono-icons/pages/subject-mono-icons.jsx]: `components/DocsBlock` - that's why it's failing to build. I'll fix it in my PR

**Pull request merged**